src/stream.iced

```diff
@@ -0,0 +1,114 @@
+stream = require('keybase-chunk-stream')
+nacl = require('keybase-nacl')
+armor = require('node-armor-x')
+msgpack = require('msgpack-lite')
+payload = require('./payload')
+header = require('./header')
+nonce = require('./nonce')
+format = require('./format')
+
+saltpack_block_len = (1024**2)
+noop = () ->
+
+class NaClEncryptStream extends stream.ChunkStream
```

Comment:
Let's add some documentation comments here too. It's not clear from the names what the difference between `NaclEncryptStream` and `EncryptStream` is supposed to be.